manifests/plugin/dynflow/params.pp

```diff
@@ -11,8 +11,5 @@
   $ssl_disabled_ciphers  = undef
   $tls_disabled_versions = undef
   $open_file_limit       = 1000000
-  $external_core         = $facts['os']['family'] ? {
-    'RedHat' => true,
-    default  => undef
-  }
+  $external_core         = false
```

Comment:
If you're changing it to default to `false`, you can also change the data type to `Boolean` since there's no way to get it back to `undef`.